src/libsyntax/ext/deriving/debug.rs

```diff
@@ -135,3 +138,19 @@ fn show_substructure(cx: &mut ExtCtxt, span: Span,
     let block = cx.block(span, stmts, Some(expr));
     cx.expr_block(block)
 }
+
+fn stmt_let_undescore(sp: Span, expr: P<ast::Expr>) -> P<ast::Stmt> {
+    let local = P(ast::Local {
```

Comment:
Ah turns out `pat_wild` exists, but unfortunately it looks like nothing exists for `local` + `decl` + `StmtDecl`